### ISSUE TITLE
[Crash]: java.lang.NullPointerException: Rendering screen

### ISSUE THREAD

**Issue by vasmark1** [FalsePattern/FalseTweaks#111]
Crash with my modpack. It'd worked well until I updated to latest version, now there is a crash before logging into a main menu. All steps of init are passed normally, but then it drops this:

```
java.lang.NullPointerException: Rendering screen
	at net.minecraft.client.renderer.texture.TextureManager.handler$zde000$falsetweaks$detectEnchantmentGlint(TextureManager.java:542)
	at net.minecraft.client.renderer.texture.TextureManager.func_110577_a(TextureManager.java)
	at net.minecraft.client.gui.GuiMainMenu.func_73968_a(GuiMainMenu.java:400)
	at net.minecraft.client.gui.GuiMainMenu.func_73971_c(GuiMainMenu.java:434)
	at net.minecraft.client.gui.GuiMainMenu.func_73863_a(GuiMainMenu.java:461)
	at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1061)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1001)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

[crash-2023-05-20_21.36.13-client.txt](https://github.com/FalsePattern/FalseTweaks/files/11523007/crash-2023-05-20_21.36.13-client.txt)
[latest.log](https://github.com/FalsePattern/FalseTweaks/files/11523008/latest.log)
[2023-05-20-7.log](https://github.com/FalsePattern/FalseTweaks/files/11523018/2023-05-20-7.log)

**Comment by FalsePattern** [comment]
Fixed in 2.5.3

**Comment by vasmark1** [comment]
Confirmed. That issue is fixed, thanks! 

But even now I cannot login because of something like this, but I don't think it's problem in your mods. 

```
Description: Registering texture

java.lang.NullPointerException: Registering texture
	at net.minecraft.client.resources.SimpleReloadableResourceManager.func_110536_a(SimpleReloadableResourceManager.java:63)
	at net.minecraft.client.renderer.texture.SimpleTexture.func_110551_a(SourceFile:29)
	at net.minecraft.client.renderer.texture.TextureManager.func_110579_a(TextureManager.java:89)
	at net.minecraft.client.renderer.texture.TextureManager.func_110577_a(TextureManager.java:45)
	at net.minecraft.client.gui.GuiMainMenu.func_73968_a(GuiMainMenu.java:400)
	at net.minecraft.client.gui.GuiMainMenu.func_73971_c(GuiMainMenu.java:434)
	at net.minecraft.client.gui.GuiMainMenu.func_73863_a(GuiMainMenu.java:461)
	at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1061)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1001)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

```
